addons/l10n_kh_pos/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
```suggestion
```
Same for the other places, this is useless nowadays and I believe I saw that they want to add a CI style to avoid these